package.json

```diff
@@ -1,42 +1,46 @@
 {
-  "name":     "taskcluster-auth",
-  "version":  "0.1.0",
-  "private":  true,
-  "main":     "node bin/server.js production",
+  "name": "taskcluster-auth",
+  "version": "0.1.0",
+  "private": true,
+  "main": "node bin/server.js production",
   "scripts": {
-    "test":   "test/runtests.sh"
+    "test": "test/runtests.sh"
   },
   "repository": {
-    "type":   "git",
-    "url":    "https://github.com/taskcluster/taskcluster-auth.git"
+    "type": "git",
+    "url": "https://github.com/taskcluster/taskcluster-auth.git"
   },
   "dependencies": {
-    "aws-sdk":                          "^2.2.0",
-    "aws-sdk-promise":                  "^0.0.2",
-    "azure-entities":                   "^0.9.0",
-    "promise":                          "^7.0.4",
-    "lodash":                           "^3.9.3",
-    "debug":                            "^2.2.0",
-    "slugid":                           "^1.1.0",
-    "taskcluster-base":                 "^0.10.2",
-    "body-parser":                      "^1.12.4",
-    "taskcluster-client":               "^0.23.9",
-    "taskcluster-lib-loader":           "^0.1.1",
-    "taskcluster-lib-app":              "^0.8.10",
-    "express":                          "^4.12.4",
-    "babel":                            "^5.8.23",
-    "kexec":                            "^1.2.0",
-    "fast-azure-storage":               "^0.3.2",
-    "hawk":                             "2.3.0"
+    "aws-sdk": "^2.2.0",
```

Comment:
But what about all my awesome indentation :(

(don't worry I won't fight you... I know I've lost @jhford won this battle over me a while ago)